Virtusize/Sources/VirtusizeExports.swift

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
By using **@_export import**, there are two benefits:
1. When a client imports Virtusize, they can also use the public classes and methods from VirtusizeCore and VirtusizeAuth for the app.
2. Ref: https://davedelong.com/blog/2018/01/19/simplifying-swift-framework-development/
  > @_exported will make an import-ed module visible to the entire module into which its been imported. This means you don’t have to import Dependency in every file you need it. You just @_exported that dependency once, and you’re good to go in any file in that module.